### DISCUSSION TITLE
Artifacts to test KMP support?

### DISCUSSION THREAD

**Discussion by manosbatsis** [dropbox/Store#456] [Q&A]
I've seen ongoing work on KMP support taking place here and there, was wondering if this is a good time to test in our app and provide feedback. We're far from production ourselves so we're probably in a good place for this, assuming the project distributes kmp artifacts for our Gradle dependencies (only Android and JVM for now).

**Comment by pikzen** [comment]
@matt-ramotar I see you've been working on making publishing work on the store5/ branch. Do you expect to publish a test artifact somewhere soon(ish), or would we be better off testing it out locally by building and pushing to an internal artifactory ?

**Comment by digitalbuddha** [comment]
Since the API is not stable, building from source makes most sense. I would recommend waiting for this pr to land 
https://github.com/MobileNativeFoundation/Store/pull/472

This should be pretty close to the final shape of things and we'd love to start getting feedback.
